pkg/cmd/server/defaults.go

```diff
@@ -47,8 +47,8 @@ func ServeExample(programName string) string {
 func DefaultPreRunE(programName string) cobrautil.CobraRunFunc {
 	return cobrautil.CommandStack(
 		cobrautil.SyncViperPreRunE(programName),
-		cobrautil.ZeroLogPreRunE("log", zerolog.InfoLevel),
-		cobrautil.OpenTelemetryPreRunE("otel", zerolog.InfoLevel),
+		cobrautil.ZeroLogRunE("log", zerolog.InfoLevel),
```

Comment:
Are these supposed to be `RunE`?

Reply:
Yep, it's up to you to decide whether it "becomes a PreRunE" if you assign it as a PreRun.